### DISCUSSION TITLE
Clan, tribe, home town, etc.

### DISCUSSION THREAD

**Discussion by WGroleau** [FamilySearch/GEDCOM#288] [Ideas]
I've often seen something like 
`2 PLAC of some_place`
or
`2 PLAC of some_ethnic_group`
or
`1 NAME Robin /of Locksley/`
but I think it would be useful to have some structured way to indicate clan or tribe or some such thing instead of the first example.  Often, it is used under BIRT when it should have been under RESI.

I've seen the first example used (also "son of") when it is how the person was known when surnames didn't exist.  That should be a NOTE or PHRASE unless some more specific tag can be introduced.

**Comment by dthaler** [comment]
Yeah, I've seen those to be very common as well, especially for use with apps/websites that don't support RESI.
One could convert them to RESI but if the receiving application doesn't support RESI then the info would be lost.
BIRT.PLAC and NAME are things that everything supports and so I understand why they are used.

**Comment by tychonievich** [comment]
GEDCOM-X has the RESI-like case as http://gedcomx.org/Heimat and tribe/clan with their own attribute types, all being discussed as part of roughly a hundred new events and attributes in #117.
